android/src/main/java/com/swmansion/rnscreens/ScreenContainer.kt

```diff
@@ -103,6 +103,7 @@ open class ScreenContainer<T : ScreenFragment>(context: Context?) : ViewGroup(co
     open fun removeScreenAt(index: Int) {
         mScreenFragments[index].screen.container = null
         mScreenFragments.removeAt(index)
+
```

Comment:
Looks like a leftover

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/1802/commits/246d3a555b18a3ab5416851ad86a92c1cd683deb.